backend/Onied/Support/Support/Services/ChatService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AutoMapper;
+using Support.Abstractions;
+using Support.Data.Abstractions;
+using Support.Dtos.Chat.GetChat.Response;
+using Support.Exceptions;
+
+namespace Support.Services;
+
+public class ChatService(
+    IChatRepository chatRepository,
+    IAuthorizationSupportUserService authorizationSupportUserService,
+    ILogger<ChatService> logger,
+    IMapper mapper) : IChatService
+{
+    public async Task<GetChatResponseDto> GetUserChat(Guid? userId)
+    {
+        if (userId is null)
+        {
+            logger.LogWarning("User ID is null. Cannot retrieve chat");
+            throw new BadRequestException("userId query parameter cannot be null");
+        }
+
+        var chat = await chatRepository.GetWithSupportAndMessagesByUserIdAsync((Guid)userId);
+        if (chat is null)
+        {
+            logger.LogWarning("No chat found for user ID: {UserId}. The chat has not yet been created", userId);
+            throw new BadRequestException("The chat has not yet been created for this user");
```

Comment:
No chat found, но при этом возвращаем badrequest. Мне кажется логичнее было бы 404.
```suggestion
            logger.LogWarning("No chat found for user ID: {UserId}. The chat has not yet been created", userId);
            throw new NotFoundException("The chat has not yet been created for this user");
```